examples/import-database-flyway-maven/pom.xml

```diff
@@ -0,0 +1,151 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Licensed to the Apache Software Foundation (ASF) under one or more
+    contributor license agreements.  See the NOTICE file distributed with
+    this work for additional information regarding copyright ownership.
+    The ASF licenses this file to You under the Apache License, Version 2.0
+    (the "License"); you may not use this file except in compliance with
+    the License.  You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+-->
+<!-- $Rev$ $Date$ -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+	xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>org.superbiz</groupId>
+	<artifactId>import-database-flyway-maven</artifactId>
+	<version>10.0.0-M3-SNAPSHOT</version>
+	<packaging>war</packaging>
+	<name>TomEE :: Examples :: Import Database By Flyway By Maven Plugin</name>
+	<properties>
+		<version.jakartaee-api>10.0-M2</version.jakartaee-api>
+		<version.arquillian>1.9.1.Final</version.arquillian>
+		<tomee.version>10.0.0-M3-SNAPSHOT</tomee.version>
+	</properties>
+	<dependencies>
+		<dependency>
```

Comment:
do we need these dependencies for this example? dont think so

Reply:
I saw the problem, and i am fixing it.